From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Wed, 2 Mar 2016 18:26:15 +0100
Subject: x86/generic: enable pata_atiixp driver

This enables booting from devices that use an ATI PATA controller for
the boot device, such as the embedded CF cards in Fujitsu-Siemens Futro
thin-clients.

Signed-off-by: Felix Kaechele <heffer@fedoraproject.org>

Backport of r48861

diff --git a/target/linux/x86/generic/config-default b/target/linux/x86/generic/config-default
index eb4b357..9e963bf 100644
--- a/target/linux/x86/generic/config-default
+++ b/target/linux/x86/generic/config-default
@@ -153,6 +153,7 @@ CONFIG_NLS=y
 CONFIG_NO_HZ=y
 # CONFIG_PANASONIC_LAPTOP is not set
 CONFIG_PATA_AMD=y
+CONFIG_PATA_ATIIXP=y
 CONFIG_PATA_LEGACY=y
 CONFIG_PATA_MPIIX=y
 CONFIG_PATA_OLDPIIX=y
